etc/config/cuda.amazon.properties

```diff
@@ -1,7 +1,10 @@
 compilers=&nvcc:&cuclang
 defaultCompiler=nvcc92
-supportsBinary=false
+supportsBinary=true
+supportsExecute=false
+rpathFlag=-L # WAR, really need `-Xcompiler "-Wl,-rpath=<path>"`, but not required because supportsExecute=false
```

Comment:
rpathFlag should be in the nvcc group, it doesn't really work for Clang

Reply:
Thanks for pointing this out, fixed.